hype-submitter/src/main/java/com/spotify/hype/Example.java

```diff
@@ -49,31 +48,34 @@ public static void main(String[] args) throws Exception {
           .collect(Collectors.toList());
     };
 
-    final RunEnvironment environment = environment("us.gcr.io/datawhere-test/hype-runner:5")
+    String image = "us.gcr.io/datawhere-test/hype-runner:5";
```

Comment:
No, good point, I was thinking of nuking it in favor of a scala example. @rouzwawi do you agree?